builder/dockerfile/parser/testfiles/escape-nonewline/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# escape = ``
```

Comment:
I don't disagree with you in the slightest. Just commenting on why it was as it was - I know that bit came up in one of the many conversations when introducing the feature 😇